ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -152,3 +152,45 @@ def test_numpy_normal(
         scale=scale,
         size=size,
     )
+
+
+# poisson
+@handle_frontend_test(
+    fn_tree="numpy.random.poisson",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=st.tuples(
+            st.integers(min_value=1, max_value=2)
+        ),
+        min_value=1,
+        max_value=100
+    ),
+
+    size=st.tuples(
+        st.integers(min_value=1, max_value=10), st.integers(min_value=2, max_value=2)
+    ),
+)
+def test_numpy_poisson(
+    dtype_and_x,
+    size,
+    as_variable,
+    num_positional_args,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        lam=x,
```

Comment:
given that it's only one array, we can probably extract it changing this to `lam=x[0]`

Reply:
Thank you for the correction! Just fixed the mistake, please let me know if there's anything else I need to do :)